### PR TITLE
Spring Cleaning

### PR DESCRIPTION
**Pull request by joebell93**
## Description

A general tidy-up of the project.

- [x] Align `@else` statements
- [x] Fix spelling mistakes
- [x] Align Spinner's color value with the color palette
- [x] Remove duplicated inline documentation and refer to `docs/`.
- [x] Remove verbose documentaion
- [x] Manual linting

## Related Issue

#412 

## Motivation and Context

Enhance developer experience (but mostly because these things were bugging me)

## How Has This Been Tested?

All tests pass, but generally, things should remain unaffected.

## Types of Changes

- [x] Internal *(framework-only)*
- [ ] Bug Fix *(non-breaking change which fixes an issue)*
- [ ] New Feature *(non-breaking change which adds functionality)*
- [ ] Breaking change *(fix or feature that would cause existing functionality to change)*

## Checklist

- [x] All **design and code** conforms to the [Design Contribution Guidelines](https://github.com/sky-uk/toolkit/blob/develop/CONTRIBUTING.md#design-contributions).
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [x] I have added instructions on how to test my changes.

### CODE REVIEW

File: packages/sky-toolkit-ui/components/_accordion.scss [hunk 1]
```diff
@@ -89,30 +85,28 @@ $accordion-icon-offset: $accordion-spacing * 2 + $accordion-icon-size;
...
-    pointer-events: none; /* [4] */
-    -ms-transform: translateY(-50%) rotate(0); /* [5] */
-    transform: translateY(-50%) rotate(0); /* [5] */
-    transition: transform $accordion-animation-speed ease; /* [6] */
+    background-repeat: no-repeat; /* [2] */
+    background-size: contain; /* [2] */
+    background-position: center center; /* [2] */
+    pointer-events: none; /* [3] */
+    -ms-transform: translateY(-50%) rotate(0); /* [4] */
+    transform: translateY(-40%) rotate(0); /* [4] */
```
Comment (lukewhitehouse) [inline packages/sky-toolkit-ui/components/_accordion.scss]: This changed from `-50%` to `-40%` on purpose?

Comment (joebell93) [reply packages/sky-toolkit-ui/components/_accordion.scss]: _hides in the corner_

File: packages/sky-toolkit-core/tools/_divider.scss [hunk 2]
```diff
@@ -2,11 +2,13 @@
 // TOOLS / #DIVIDER
 // =============================================================================
 
-// _divider-gradient-color-points()
+// Private / Framework Only
```
Comment (StefanMcCready) [inline packages/sky-toolkit-core/tools/_divider.scss]: I assume we need this for public repo reasons?

Comment (joebell93) [reply packages/sky-toolkit-core/tools/_divider.scss]: Nope, this just means functions/mixins underneath are only for use within this framework

Comment (StefanMcCready) [reply packages/sky-toolkit-core/tools/_divider.scss]: nice

**Pull request merged**